### DISCUSSION TITLE
Loadstep definition - BC - mechanical : which YAML syntax to use to define rotation ?

### DISCUSSION THREAD

**Discussion by salettid** [damask-multiphysics/DAMASK#28] [Q&A] [answered]
In the online documentation, grid solver page, the R parameter related to mechanical boundary condition is described such as :

R: rotation axis and angle (in degrees) from grid to load frame coordinate system (defaults to no rotation, i.e. load frame coincides with grid coordinates)

What is the syntax to provide ? A tuple such as, for intance,  ('z', 30.0 ) ? or an Euler sequence angle in a list such as [30.0, 0.0,0.0] ?

Thanks 
Dominique

**Comment by sharanroongta** [comment, marked as answer]
It requires the rotation axis and the angle. R: [0,0,1,30] # rotation of 30 degree around a axis

**Reply by eisenlohr** [reply]
We sharpened the language of the documentation and updated the website to include a corresponding example.

**Comment by salettid** [comment]
Great. Thanks a lot.
